modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZonesUtil.java

```diff
@@ -74,12 +76,24 @@ public class DistributionZonesUtil {
     /** Key prefix for zone's data nodes history. */
     public static final String DISTRIBUTION_ZONE_DATA_NODES_HISTORY_PREFIX = DISTRIBUTION_ZONE_DATA_NODES_PREFIX + "history.";
 
+    /** Key prefix for zone's data nodes history, in {@link ByteArray} representation. */
+    public static final ByteArray DISTRIBUTION_ZONE_DATA_NODES_HISTORY_PREFIX_BYTES =
```

Comment:
Why do you use `ByteArray` and not `byte[]` here?

Reply:
fixed